src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Area {
```

Comment:
If Figure class implements some interfaces then all its children implements these interfaces automatically
```suggestion
public class Circle extends Figure {
```